src/bullMQOtel.ts

```diff
@@ -91,8 +94,18 @@ export class BullMQOtel implements Telemetry<OtelContext> {
   tracer: BullMQOtelTracer;
   contextManager: BullMQOTelContextManager;
 
-  constructor(tracerName: string, version?: string) {
-    this.tracer = new BullMQOtelTracer(trace.getTracer(tracerName, version));
-    this.contextManager = new BullMQOTelContextManager();
+  meter: OtelMeter;
+
+  constructor(telemetry: Configuration) {
```

Comment:
For every implementation of the telemetry interface we can have any specific constructor that makes sense for a particular implementation. For example in this Otel constructor we could have settings specific for Otel, this works because the implementation is instantiated and passed to the Queue and Worker constructors that expect an instance of an object implementing the Telemetry object, it is irrelevant how this object was created. So having Configuration here taken from BullMQ does not make any sense...